frontend/src/pages/NewRun.test.tsx

```diff
@@ -364,7 +382,9 @@ describe('NewRun', () => {
     expect(tree.state()).toHaveProperty('pipeline', MOCK_PIPELINE);
     expect(tree.state()).toHaveProperty('pipelineName', MOCK_PIPELINE.name);
     expect(tree.state()).toHaveProperty('pipelineVersion', MOCK_PIPELINE_VERSION);
-    expect(tree.state()).toHaveProperty('errorMessage', 'Run name is required');
+    expect((tree.state() as any).runName).toMatch(
+      /Run_of_\(original mock pipeline version name\)_.*/,
```

Comment:
IMO, what do you think about `Run xxxx of (original mock pipeline version name)`?

There's no need for underscore and the order looks more like english?

Reply:
Done